addons/project/static/src/components/project_task_state_selection/project_task_state_selection.xml

```diff
@@ -7,19 +7,19 @@
         <!-- Waiting state button -->
         <xpath expr="//t[@t-if='isReadonly']" position="after">
             <t t-elif="currentValue == '04_waiting_normal' and ['kanban', 'list'].includes(this.props.viewType)">
-                <button class="d-flex align-items-center btn fw-normal p-0 justify-content-center " title="This task is blocked by another unfinished task">
+                <button class="d-flex align-items-center btn fw-normal p-0 justify-content-center " title="This task is blocked by another unfinished task" t-att-class="{'o_task_state_list_view': this.props.viewType === 'list'}">
                     <i class="fa fa-lg fa-hourglass-o"></i>
                 </button>
             </t>
         </xpath>
         <!-- The toggle mark as done button -->
         <xpath expr="//t[@t-if='isReadonly']" position="after">
             <t t-elif="isToggleMode and currentValue == '01_in_progress'">
-                <button t-if="['kanban', 'list'].includes(this.props.viewType) or this.env.isSmall" class="d-flex align-items-center btn fw-normal p-0" t-on-click.stop="toggleState">
-                    <i t-attf-class="{{ stateIcon(currentValue) }} {{ statusColor(currentValue) }}"></i>
+                <button t-if="['kanban', 'list'].includes(this.props.viewType) or this.env.isSmall" class="d-flex align-items-center btn fw-normal p-0" t-on-click.stop="toggleState" t-att-class="{'o_task_state_list_view': this.props.viewType === 'list'}">
```

Comment:
`this.props.viewType === 'list/kanban'` could be a function? `isView('list')`? What do you think?

Reply:
Also, doesn't it work without "this"?

Reply:
Done with the function